README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://LinaDo1609.github.io/layout_calendar/)
```

Comment:
Do not forget to add these links to the PR description

Reply:
> Do not forget to add these links to the PR description

i have added them

Reply:
> Do not forget to add these links to the PR description

i didn't add them just because this task doesn't have test. that's why i thought it isn't necessary